src/App.tsx

```diff
@@ -28,36 +27,37 @@ const data = {
   ],
 };
 
+const Home = () => (
+  <div>
+    <a  style={{marginLeft: '1000px'}} href='/module/0/emulator/0'>Visit Emulator</a> <br />
+    <a href='/module'>Visit Courses</a>
+  </div>
+);
+
 function App() {
   return (
     <Router>
-      <Switch>
-        <Route exact path="/">
-          <Layout>
-            <ModuleList />
-          </Layout>
-        </Route>
-        <Route exact path="/module/:moduleId/quiz/:quizId">
-          <Layout>
-            <Quiz />
-          </Layout>
-        </Route>
-        <Route exact path="/module/:moduleId/article/:articleId">
-          <Layout>
-            <Article />
-          </Layout>
-        </Route>
-        <Route path="/module/:moduleId/emulator/:emulatorId">
-          <Layout>
-           <Emulator panelData={data} />
-          </Layout>
-        </Route>
-        <Route path="*">
-          <Layout>
-            <h1>Page not found...</h1>
-          </Layout>
-        </Route>
-      </Switch>
+      <Header />
+          <Switch>
+            <Route exact path='/'>
+              <Home />
+            </Route>
+            <Route exact path="/module">
+                <ModuleList />
+            </Route>
+            <Route exact path="/module/:moduleId/quiz/:quizId">
+                <Quiz />
+            </Route>
+            <Route exact path="/module/:moduleId/article/:articleId">
+                <Article />
+            </Route>
+            <Route path="/module/:moduleId/emulator/:emulatorId">
+              <Emulator panelData={data} />
+            </Route>
+            <Route path="*">
+                <h1>Page not found...</h1>
+            </Route>
+          </Switch>
```

Comment:
Can you un-indent lines 41-60 so that it matches the indentation level of line 40?